resources/Homestead.yaml

```diff
@@ -21,6 +21,7 @@ databases:
     - homestead
 
 features:
+    - mysql: false
```

Comment:
Should this have been `true`?

Reply:
The value is only evaluated if you've enabled the backup option. in which case you should enable MySQL if you're using it.

Reply:
I ask because in the same PR the mysql key's value was changed to true in `Homestead.json` - if the two files are not supposed to be the same, then I apologize for the noise.

Reply:
Not a *huge* deal. IMHO it should default to false, but I can see the reasoning for it to default to true as well.